lldb/source/Host/CMakeLists.txt

```diff
@@ -190,3 +190,8 @@ add_lldb_library(lldbHost NO_PLUGIN_DEPENDENCIES
     ${LLDB_LIBEDIT_LIBS}
   )
 
+if (LLDB_ENABLE_LIBXML2 AND LLVM_USE_STATIC_LIBXML2)
+  target_link_libraries(lldbHost PRIVATE ${LIBXML2_LIBRARIES})
+  target_include_directories(lldbHost PUBLIC ${LIBXML2_INCLUDE_DIRS})
+  add_dependencies(lldbHost libxml2)
+endif()
```

Comment:
The goal of using the interface libraries (like `LibXml2::LibXml2`) is to simplify our CMake code where we don't need to manually update the list include directories and link libraries and can instead depend on a single target, but this negates that benefit, even worse now we have to both depend on the interface target and manually update the list include directories and link libraries everywhere where we depend on libxml2.